web/package.json

```diff
@@ -10,28 +10,28 @@
     "build": "vite build",
     "preview": "vite preview",
     "check": "svelte-check --tsconfig ./tsconfig.json",
-    "fmt": "npx prettier --write *.html src/**",
+    "fmt": "npx prettier --write *.js *.ts *.json *.html src/**",
     "list-tests": "./public/list_tests.sh"
   },
   "devDependencies": {
     "@ianvs/prettier-plugin-sort-imports": "^4.0.2",
     "@sveltejs/vite-plugin-svelte": "^2.4.2",
-    "@tsconfig/svelte": "^4.0.1",
+    "@tsconfig/svelte": "^5.0.2",
     "@types/mapbox__mapbox-gl-draw": "^1.4.0",
-    "prettier": "^2.8.8",
-    "prettier-plugin-svelte": "^2.10.1",
+    "prettier": "^3.2.5",
+    "prettier-plugin-svelte": "^3.2.2",
     "svelte": "^4.0.0",
     "svelte-check": "^3.4.4",
     "tslib": "^2.6.0",
     "typescript": "^5.1.3",
     "vite": "^4.3.9",
-    "vite-plugin-wasm-pack": "0.1.11"
+    "vite-plugin-wasm-pack": "^0.1.12"
   },
   "dependencies": {
     "@mapbox/mapbox-gl-draw": "^1.4.1",
     "@maptiler/geocoding-control": "^1.1.1",
     "@turf/bbox": "^6.5.0",
-    "maplibre-gl": "^3.1.0",
-    "svelte-maplibre": "^0.3.1"
+    "maplibre-gl": "^4.1.0",
+    "svelte-maplibre": "^0.8.2"
```

Comment:
Amusingly this dependency isn't used at all right now; I don't remember why I added it. I'm using this in all my other projects, and it makes working with maplibre so much nicer. `Layer.svelte` in this repo can mostly be replaced by it. I can catch up on some refactoring in the web app eventually